getting-started-intellij-track/getting-started-with-scala-in-intellij.md

```diff
@@ -36,7 +36,7 @@ take a few minutes but subsequent projects can use the same SDK.
 ## Writing code
 
 1. On the **Project** pane on the left, right-click `src` and select
-**New** => **Scala class**.
+**New** => **Scala class**. If you don't see **Scala class**, right-click on **HelloWorld** and click on **Add Framework Support...**, select **Scala** and proceed. If you see **Error: library is not specified**, you can either click donload button, or select the library path manually.
```

Comment:
There's a typo in "download".

Reply:
```suggestion
**New** => **Scala class**. If you don't see **Scala class**, right-click on **HelloWorld** and click on **Add Framework Support...**, select **Scala** and proceed. If you see **Error: library is not specified**, you can either click download button, or select the library path manually.
```

Reply:
Fixed typo.
@dwijnand IMHO putting this change in prerequisite section will be better because any new installation will need this step.